apps/kyc-server/src/services/__tests__/kyc-reviews.service.test.ts

```
@@ -0,0 +1,67 @@
+import { describe, it, expect, beforeEach, afterEach } from 'bun:test';
+import { KycReviewsService } from '../kyc-reviews.service';
+import type { CreateKycReviewInput } from '@kindfi/shared/types/kyc-reviews';
+
+describe('KycReviewsService', () => {
+  let service: KycReviewsService;
+  let testUserId: string;
+
+  beforeEach(() => {
+    service = new KycReviewsService();
+    testUserId = 'test-user-' + Date.now();
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)